case/src/main/kotlin/com/ritense/case_/service/CaseWidgetTabService.kt

```diff
@@ -43,4 +50,37 @@ class CaseWidgetTabService(
         return caseWidgetTabRepository.findByIdOrNull(CaseTabId(caseDefinitionName, key))
             ?.let { CaseWidgetTabDto.of(it) }
     }
+
+    fun updateWidgetTab(tabDto: CaseWidgetTabDto): CaseWidgetTabDto? {
+        denyAuthorization()
+
+        val caseWidgetTab = caseWidgetTabRepository.findById(CaseTabId(tabDto.caseDefinitionName, tabDto.key))
```

Comment:
Could've used `findByIdOrNull` here to keep it more Kotlin-style, but this works too.